force-app/main/default/classes/userProfileHelper.cls

```diff
@@ -0,0 +1,7 @@
+public with sharing class userProfileHelper {
+    @AuraEnabled
+    public static Map<String,Object> getUserProfile() {
+        List<Profile> myProfile = [SELECT Id, Name FROM Profile WHERE Id=:userinfo.getProfileId() LIMIT 1];
```

Comment:
You should perform [field security checks](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_perms_enforcing.htm) on `Profile.Id` and `Profile.Name` before attempting to execute the SOQL query.

Reply:
Thanks, I had no idea!